pkg/skaffold/build/local/util_windows.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2018 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package local
+
+import (
+	"os/exec"
+	"path/filepath"
+)
+
+// Maven and Gradle projects often provide a wrapper to ensure a particular
+// builder version is used.  This function tries to resolve a wrapper
+// or otherwise resolves the builder executable.
+func findBuilder(builderExecutable string, wrapperScriptName string, workspace string) ([]string, error) {
+	wrapperFile := filepath.Join(workspace, wrapperScriptName)
+	if isFile(wrapperFile) {
+		path, err := filepath.Abs(wrapperFile)
+		if err != nil {
+			return nil, err
+		}
+		return []string{path}, nil
+	}
+	if cmdFile := wrapperFile + ".cmd"; isFile(cmdFile) {
```

Comment:
if these are windows only, the way to do OS specific functions in Go is to use build constraints

https://golang.org/pkg/go/build/#hdr-Build_Constraints

You can see an example of this https://github.com/kubernetes/minikube/tree/master/pkg/minikube/cluster

Also you can rewrite this logic to be something like
```
for _, f := range []string{"", ".cmd", ".bat"} {
 wf := wrapperFile + f
 ...logic, maybe return early
}
```

Reply:
Is using an explicit `// +build` preferable to using the `* _GOOS` file name convention?

Reply:
`_GOOS` is preferable if its OS based